[OSDOCS-17003] IBM Z callout replacement: replace AsciiDoc callouts with definition … (#116250)
openshift/openshift-docs: modules/ibm-z-configure-devices-mco.adoc

Reviewer: All in all, this was a really solid way of removing the smaller headings without causing a ton of changes to the existing structure. Nicely done!

:_mod-docs-content-type: PROCEDURE
[id="configure-additional-devices-using-mco_{context}"]
= Configuring additional devices by using the Machine Config Operator

[role="_abstract"]

> Tasks in this section describe how to use features of the Machine Config Operator (MCO) to configure additional devices in an {ibm-z-name} or {ibm-linuxone-name} environment. Configuring devices with the MCO is persistent but only allows specific configurations for compute nodes. MCO does not allow control plane nodes to have different configurations.

Reviewer: A few things here:
1. Abstracts shouldn't be self referential (e.g. saying "this section", "this procedure", "the following..."). So in cases where the pre-existing intro is self referential, you have to find a way to either rewrite the intro, or move it to its own paragraph and use a brand new paragraph as the abstract.
2. For abstracts, the 300 character limit is strict (apparently files in AEM will literally refuse to save if the abstract is over the limit). This one is 341 characters, with attributes resolved. So if you can, try to trim them down now. One easy way is, if the last sentence or two really isn't critical to the core of explaining the content, just move it to its own paragraph

This rewrite satisfies both things, making the abstract 299 characters haha:

```suggestion
You can use the Machine Config Operator (MCO) to configure additional devices in an {ibm-z-name} or {ibm-linuxone-name} environment. Configuring devices with the MCO is persistent but only allows specific configurations for compute nodes. MCO does not allow control plane nodes to have different configurations.
```

If you want to play it extra safe, then to be honest it's fine if only the first sentence here is part of the abstract, and the rest are in a following paragraph

Author: I love challenges lets keep it with the 299 characters 🤣

     storage:
       files:
       - contents:

>            source: <base64_data_uri>

Author: In case someone is wondering why I changed this. IBM enforced Vault Radar scans on block secrets last week and I had no chance getting the file updated without changing that string. The commit was entirely rejected.

Reviewer: No worries, in this case the PR is QE reviewed so as long as they signed off on the change, it's alright.

I don't fully understand the nature of the issue but it sounds like you were forced to make this change? If so, fair enough for this PR.
